alibi_detect/utils/pytorch/prediction.py

```diff
@@ -54,11 +54,11 @@ def predict_batch(x: Union[list, np.ndarray, torch.Tensor], model: Union[Callabl
                     preds = tuple([] for _ in range(len(preds_tmp)))
                     return_list = isinstance(preds_tmp, list)
                 for j, p in enumerate(preds_tmp):
-                    if device.type == 'cuda' and isinstance(p, torch.Tensor):
+                    if device.type == 'cuda' and isinstance(p, torch.Tensor):  # type: ignore
```

Comment:
What `mypy` error is raised here? It would be good to add an error code specific type ignore so that it is clear what is being ignored `# type: ignore[<error-code>]`. (this is something we're trying to do more going forward, as it leads to more readable code).